nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -124,3 +124,5 @@ In addition to numerous new and upgraded packages, this release has the followin
 - `fetchFromSourcehut` now allows fetching repositories recursively
   using `fetchgit` or `fetchhg` if the argument `fetchSubmodules`
   is set to `true`.
+
+- The option `services.thelounge.plugins` has been added to allow installing plugins for The Lounge.
```

Comment:
Discoverability of these new plugins is a concern and might need clarification here as well.